annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -74,29 +81,52 @@ fun main(argv: Array<String>) {
     val repoToPushTo = params.getString("repoToPushTo")
     val localWorkingCopyDirectory = File(params.getString("localWorkingCopyDirectory"))
 
+    val exportedAnnotationRoot = params.getCreatableDirectory("exportedAnnotationRoot")
+
+    // Build params for exporting the annotations
     val exportAnnotationsParamsBuilder = Parameters.builder()
     exportAnnotationsParamsBuilder.set("inceptionUrl", params.getString("inceptionUrl"))
     exportAnnotationsParamsBuilder.set("inceptionUsername", params.getString("inceptionUsername"))
     exportAnnotationsParamsBuilder.set("inceptionPassword", params.getString("inceptionPassword"))
-    exportAnnotationsParamsBuilder.set("exportedAnnotationRoot","$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
+    exportAnnotationsParamsBuilder.set("exportedAnnotationRoot", "$exportedAnnotationRoot")
     val exportAnnotationsParams = exportAnnotationsParamsBuilder.build()
 
+    // Build params for restoring the original text
     val restoreJsonParams = if (params.getOptionalBoolean("restoreJson").or(false)) {
         Parameters.builder()
-                .set("indexDirectory", params.getString("indexDirectory"))
-                .set("gigawordDataDirectory", params.getString("gigawordDataDirectory"))
-                .set("inputJsonDirectory", "$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
-                .set("restoredJsonDirectory", "$localWorkingCopyDirectory" + params.getString("restoredJsonDirectory"))
+                .set("indexDirectory", "${params.getExistingDirectory("indexDirectory")}")
```

Comment:
PRefer the more explicit:
```suggestion
                .set("indexDirectory", params.getExistingDirectory("indexDirectory").getAbsolutePath())
```

Reply:
(and similarly below)